### PR TITLE
test(plugin-meetingsV2): add sipUrl to body from sipUri

### PR DESCRIPTION
**Pull request by jonniespratley**
- Add conditional to handle adding `sipUrl` to `body`
- Add spec to test for `meetingNumber

Screen Shot 2021-12-07 at 9 10 45 AM
`

### CODE REVIEW

**Comment by jonniespratley** [comment]
Note: This is with `enabledUnifiedMeetings = true` so I am not sure how the conditional should be in the spec.

File: packages/node_modules/@webex/plugin-meetings/src/config.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default {
     enableExtmap: false,
     experimental: {
       enableMediaNegotiatedEvent: false,
-      enableUnifiedMeetings: false
+      enableUnifiedMeetings: true
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/config.js:91]: turn it off after taking a screen shot of test which passes

File: packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js [hunk 2]
```diff
@@ -65,8 +65,13 @@ skipInNode(describe)('plugin-meetings', () => {
           testUtils.delayedPromise(alice.meeting.join()),
           testUtils.waitForEvents([
             {scope: bob.webex.meetings, event: 'meeting:added', user: bob},
-            {scope: chris.webex.meetings, event: 'meeting:added', user: chris}])
-        ])));
+            {scope: chris.webex.meetings, event: 'meeting:added', user: chris}
+          ])
+        ]).then(() => {
+          const {meetingNumber} = bob.meeting.meetingInfo;
+
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js:72]: nice thanks

**Review by arun3528 — APPROVED** [review]
Add the screen shot of the test passing with the toggle

**Review by taymoork2 — APPROVED** [review]
LGTM

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Can you check why the integration tests are consistently failing?

**Review by lalli-flores — APPROVED** [review]
`plugin-meetings space-meeting alice adds x user as guest to space meeting` integration tests seem to be particularly flaky today

**Pull request merged**